site/docusaurus.config.ts

```diff
@@ -0,0 +1,150 @@
+import type * as Preset from '@docusaurus/preset-classic';
+import type { Config } from '@docusaurus/types';
+import { themes as prismThemes } from 'prism-react-renderer';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+const organizationName = 'openvinotoolkit';
+const projectName = 'openvino.genai';
+
+const config: Config = {
+  title: 'OpenVINO GenAI',
+  favicon: 'img/favicon.png',
+
+  // Set the production url of your site here
+  url: `https://${organizationName}.github.io`,
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: `/${projectName}/`,
+
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName, // Usually your GitHub org/user name.
+  projectName, // Usually your repo name.
+
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'throw',
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
+  },
+
+  presets: [
+    [
+      'classic',
+      {
+        docs: {
+          sidebarPath: './sidebars.ts',
+          // Please change this to your repo.
+          // Remove this to remove the "edit this page" links.
+          editUrl: `https://github.com/${organizationName}/${projectName}/tree/main/site`,
+        },
+        blog: false,
+        theme: {
+          customCss: './src/css/custom.css',
+        },
+      } satisfies Preset.Options,
+    ],
+  ],
+
+  themeConfig: {
+    colorMode: {
+      disableSwitch: true,
+      defaultMode: 'light',
+    },
+    navbar: {
+      title: 'OpenVINO GenAI',
+      logo: {
+        alt: 'Intel logo',
+        src: 'img/intel-logo.svg',
+      },
+      items: [
+        {
+          type: 'docSidebar',
+          sidebarId: 'gettingStartedSidebar',
+          position: 'left',
+          label: 'Documentation',
+          to: '/docs',
+        },
+        {
+          type: 'html',
+          position: 'left',
+          value: '<i style="color: #fec91b"><b>Status:</b> Work in progress</i>',
+        },
+        {
+          href: 'https://github.com/openvinotoolkit/openvino.genai',
+          label: 'GitHub',
+          position: 'right',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'OpenVINO',
+          items: [
+            {
+              label: 'OpenVINO™ Telemetry',
+              href: 'https://docs.openvino.ai/2024/about-openvino/additional-resources/telemetry.html',
```

Comment:
GenAI doesn't use telemetry. I'd delete it.

Reply:
Replace with link to base OpenVINO docs